src/getPortalSdk.ts

```diff
@@ -22,6 +50,13 @@ export const getPortalSdk = () => {
       const result = await sdk.addShare({ input });
       return result.addShare || null;
     },
+    TrackEvent: async (input: {
```

Comment:
nit: lowercase for function methods `trackEvent`

Reply:
cb1fd351549d60675574453f5bc999c7ea07ac42